packages/widget_toolkit_pin/lib/src/lib_pin_code_with_biometrics/blocs/pin_code_bloc.dart

```diff
@@ -78,45 +48,67 @@ class PinCodeBloc extends $PinCodeBloc {
     required this.pinCodeService,
     required this.enterPinWithBiometrics,
   }) {
-    checkPinCodeInStorage();
-    checkBiometricsEnabled();
-    checkAvailableBiometrics();
-    isPinCodeInSecureStorage.connect().addTo(_compositeSubscription);
-    isAuthenticatedWithBiometrics.connect().addTo(_compositeSubscription);
-    isPinCodeVerified.connect().addTo(_compositeSubscription);
-    availableBiometrics.connect().addTo(_compositeSubscription);
-    pinLength.connect().addTo(_compositeSubscription);
+    authenticated.connect().addTo(_compositeSubscription);
+    showBiometricsButton.connect().addTo(_compositeSubscription);
   }
 
   final PinBiometricsService biometricAuthenticationService;
   final PinCodeService pinCodeService;
   final String enterPinWithBiometrics;
```

Comment:
I think the name of this variable `enterPinWithBiometrics` should better be named `localizedReason` as it is in the other layers